os/linux/weston/plugin/compositor-iahwc.c

```diff
@@ -445,9 +445,11 @@ static int iahwc_output_repaint(struct weston_output *output_base,
     output->cursor_plane.y = INT32_MIN;
   }
 
+#if 0
   iahwc_output_render(output, damage);
```

Comment:
why do we need this ?

Reply:
A better descriptive name than ? i.e. DISABLE_OVERLAY ?

Reply:
Okay. And set that variable in the makefile?

Reply:
okay. I got the makefile part also ready. WIll give a PR in weston if okay.